Automattic/wp-calypso
# Add Analyzer data store and update light importer screens

#### Proposed Changes

* Introduced Analyzer `@wordpress/data` data store. The fundament store for the future analysis methods, such as photos, typographies, etc.
* Updated light importer component screens to support showing variable data.

In the meantime, a new design brings changes to the text and adds one more screen step, but the primary purpose of this PR is the Analyzer data store, so in the following PRs, we will bring final design changes.

FYI: For presenting colors, we are providing only link colors in this dev phase. (will be updated in the following PRs)

#### Testing Instructions

* Go to `/setup/importLight?siteSlug={SLUG}`
* Provide a website (example: https://deafening-tern.jurassic.ninja/)
* Check redux actions (if you have a Redux plugin, there is a nice UI to check it)

#### Screenshot
<img width="1085" alt="Screenshot 2022-06-20 at 12 55 03" src="https://user-images.githubusercontent.com/1241413/174586965-ed54828a-47cd-4165-a234-d84b542df117.png">


Related to #64774
